### PR TITLE
Add OSSEC

### PR DESCRIPTION
**Pull request by awiddersheim**
OSSEC is an Open Source Host-based Intrusion Detection System that performs log analysis, file integrity checking, policy monitoring, rootkit detection, real-time alerting and active response.

### CODE REVIEW

**Comment by n1trux** [comment]
Your description is 192 characters long. Please shorten it a little (it *should* fit in one line) and remove redundancies (OSSEC, Open Source, ...). Guide yourself by the other descriptions.

And please include a link to the OSS project page in your pull request next time.
I confirmed the OSS status this time, but I'd be faster with merging if you already included it.

Also read: [Contribution guidelines](../blob/master/CONTRIBUTING.md)

**Comment by awiddersheim** [comment]
I read the contribution guidelines. It only said `Keep descriptions short`. Nothing about only keeping things to a single line. Short is relative, and I thought the description was short but have shortened it further per your request.

 Although many of the existing descriptions do conform to your one line rule some do not. For example, Galera or Ganglia.

Finally, the link to the project page was in the description line. Did you want it in the actual pull request description? I figured the link the description was enough.

Basically, I read the [Contribution Guidlines](https://github.com/n1trux/awesome-sysadmin/blob/master/CONTRIBUTING.md) and based my pull request off of what was in that so perhaps it needs some tweaks to include some of the stuff you just mentioned to me.

**Comment by n1trux** [comment]
Apologies, thanks, merged.

>Although many of the existing descriptions do conform to your one line rule some do not. For example, Galera or Ganglia.

You are right, we have to edit Galera's bloaty description, too. With Ganglia's I'm not so sure.

After all, the contribution guidelines are just that, guidelines. But repeating the app's name and open source status (which is redundant, since all the apps should be FLOSS) is bloaty. Thanks for your input :+1:

Please note #13, too.
